app/routes.py

```diff
@@ -1,2 +1,28 @@
-from flask import Blueprint
+from flask import Blueprint, make_response, abort
+from app.models.planet import planets
 
+planet_bp = Blueprint("planet_bp", __name__, url_prefix="/planets")
+
+@planet_bp.get("")
+def get_all_planets():
+    return [planet.to_dict() for planet in planets]
+
+@planet_bp.get("/<planet_id>")
+def get_one_planet(planet_id):
+    planet = validate_planet(planet_id)
+
+    return planet.to_dict()
+
+def validate_planet(planet_id):
+    try:
+        planet_id = int(planet_id)
+    except:
+        response = {"messange" : f"planet id {planet_id} is invalid"}
+        abort(make_response(response, 400))
```

Comment:
We could avoid the need to use `try ... except` block here by specifying a converter in the URL like this:
```python
@planet_bp.get("/<planet_id>")
``` 
But in order to follow the instructions of the lesson, we kept the `try ... except` block here.